tox.ini

```diff
@@ -32,6 +32,14 @@ deps =
     -rrequirements-dev.txt
 commands = {[testenv]commands}
 
+# Test with python 3.11; optional and dev reqs (some features are not compatible with 3.12 yet)
+[testenv:py311-optional]
+basepython = python3.11
+deps =
+    -rrequirements-dev.txt
+    -rrequirements-opt.txt
+commands = {[testenv]commands}
+
```

Comment:
Is this workflow called?

Reply:
I believe so. That's how I got the coverage to pass.

Reply:
Unless I am missing something

Reply:
Oh the run tests workflow

Reply:
Yeah, "tox -e py311-optional" needs to be added to the run tests workflow and run-all-tests workflow.

Reply:
Slipped my mind. Thanks